### DISCUSSION TITLE
Can I use assimp as a third party of the Apache-2.0 license library?

### DISCUSSION THREAD

**Discussion by ShenMian** [assimp/assimp#4137] [Q&A] [answered]
[This](https://github.com/ShenMian/Graphics) is the library I am working on, using the Apache-2.0 license. I added assimp in the '[Thirdparty](https://github.com/ShenMian/Graphics/tree/main/Thirdparty)' fold as a submodule.
If I can't do this, please let me know. Thank you.

**Comment by kimkulling** [comment, marked as answer]
This is completely fine with our license. Have fun!

Kim
